controllers/ListItemController.php

```diff
@@ -17,6 +17,34 @@
 
 	class ListItemController
 	{
+		/**
+			* Swagger doc block
+			*
+			* @OA\Get(
+			* path="ListItem/get_list_items",
+			* summary="Find list items by list id",
+			* @OA\Parameter(
+			*		name="listID",
+			* 	in= "query",
+			* 	description="Identifier of the list to fetch items from",
+			* 	required=true,
+			* 	type="string",
+			* 	style="form"
+			*	),
+			* @OA\Response(
+			*		response=200,
+      *   description="successful operation",
+			*		@OA\Schema(
+ 			*   	type="array",
+ 			*  		@OA\Items(ref="#/components/schemas/Item")
+			* 	),
+			* ),
+			*	@OA\Response(
+			* 	response="401",
+			*   description="Invalid list id",

```

Comment:
The `401` status code refers to the `error_response(Translation::translate('invalid_token'), 401);` which is for the invalid token provided and not the `invalid list id`

Reply:
made the required change.